### ISSUE TITLE
Neutral NPCs Treated as Enemies, Prevent Hangaring

### ISSUE THREAD

**Issue by subbob** [DraconisCluster/DX-Issues#14]
**TL;DR**
--Neutral NPCs (Ice Haulers, Trade Stations, Faction HQs, etc) prevent Hangaring within 25km

**Full Info**
-- White NPCs, neutral, non-enemy are perceived are treated as enemies by the hangar mod
-- Thus preventing hangaring when within 25km
-- In slow zones, an Ice Hauler spawning nearby can prevent hangaring of a non-mobile station for a very long time, putting it at risk
-- IIRC on prior SDG servers (and other SE servers on which I've played), one was able to hangar a ship in the vicinity of neutral trade stations

**Why bug report it now?**
-- There have been various discussion on the Discord, player side and perhaps staff side, about this issue. I can't recall who all was involved. Some seemed surprised that this is happening and/or indicated it should not be.
-- I submitted the but report to raise awareness and get decision, whether that decision is keep as is or attempt a configuration change.

**Screenshots**
![image](https://github.com/DraconisCluster/DX-Issues/assets/5113030/ea50f760-b892-41b4-8165-0fee22576ce1)

**Comment by Terif1** [comment]
Duplicates #12, could you please add onto that one
